tests/wasm.rs

```diff
@@ -839,76 +856,87 @@ async fn test_scan_with_index() {
         exclusive: bool,         // scan start_exclusive
         expected: Vec<(&str, &str, &str)>, // expected results of the scan
     ) {
-        for use_memstore in vec![true, false] {
```

Comment:
ignore whitespace diff using https://github.com/rocicorp/repc/pull/328/files?diff=split&w=1

This test was a bit flawed when I wanted to test it using MemStore. It really made no sense but it was failing now because closing the db removes the state when we use a MemStore